packages/components/stories/SubHeaderBar.js

```diff
@@ -143,10 +144,11 @@ stories
 				subTitle="mySubTitle"
 				onGoBack={backAction}
 				componentsCenter={[componentInput]}
+				editable
 			/>
 		</div>
 	))
-	.addWithInfo('with center && right components', () => (
+	.addWithInfo('with center && right components, no edit', () => (
```

Comment:
Same remark here, please remove the `editable` from the others and change this to `with edit icon`, remove all the other specific props